python/lsst/ip/isr/linearize.py

```diff
@@ -412,9 +412,12 @@ def validate(self, detector=None, amplifier=None):
             if int(self._detectorId) != int(detector.getId()):
                 raise RuntimeError("Detector IDs don't match: %s != %s" %
                                    (int(self._detectorId), int(detector.getId())))
-            if self._detectorSerial != detector.getSerial():
-                raise RuntimeError("Detector serial numbers don't match: %s != %s" %
-                                   (self._detectorSerial, detector.getSerial()))
+            # DM-38778: This check fails on LATISS due to an error in the
```

Comment:
Consider prefacing this and below note with `TODO: Uncomment when DM-38778 is complete...`, if you find TODO tags helpful.